shared/featureconfig/config.go

```diff
@@ -29,8 +29,10 @@ type FeatureFlagConfig struct {
 	EnableComputeStateRoot       bool // EnableComputeStateRoot implementation on server side.
 	EnableCrosslinks             bool // EnableCrosslinks in epoch processing.
 	EnableCheckBlockStateRoot    bool // EnableCheckBlockStateRoot in block processing.
-	EnableHistoricalStatePruning bool // EnableHistoricalStatePruning when updatifinalized states.
+	EnableHistoricalStatePruning bool // EnableHistoricalStatePruning when updating finalized states.
 	EnableCommitteesCache        bool // EnableCommitteesCache for state transition.
+	EnableBlockAncestorCache     bool //EnableBlockAncestorCache for fork choice optimization.
+	EnableBlock                  bool // EnableCommitteesCache for state transition.
```

Comment:
Enable block? what is this? The comment seems outdated too

Reply:
oops how did that happen.. thx fo catching!